src/components/Layout/types.ts

```diff
@@ -0,0 +1,6 @@
+import { ReactNode } from 'react';
+
+export type LayoutProps = {
+    // children?: JSX.Element | JSX.Element[];
+    children: ReactNode;
+};
```

Comment:
React has a built in type called `PropsWithChildren` (React.PropsWithChildren) which you can use and it will automatically type the children prop. It's a generic type which means if the component needs more Props besides children prop, you can do this:
```
interface Props {
     additionalProp: string;
     isOpen?: boolean;
}
PropsWithChildren<Props>
```
If you don't have any other props at the moment just put `PropsWithChildren<unknown>`

Reply:
> React has a built in type called `PropsWithChildren` (React.PropsWithChildren) which you can use and it will automatically type the children prop. It's a generic which means if the component needs more Props besides children prop, you can do this:
> 
> ```
> interface Props {
>      additionalProp: string;
>      isOpen?: boolean;
> }
> PropsWithChildren<Props>
> ```
> 
> If you don't have any other props at the moment just put `PropsWithChildren<unknown>`

It should be ok now